src/lib/incentives-calculation.ts

```diff
@@ -381,13 +381,9 @@ export default function calculateIncentives(
   const authorities: AuthoritiesById = {};
   if (stateAuthorities) {
     incentives.forEach(i => {
-      if (
-        'authority' in i &&
-        i.authority &&
-        (i.authority_type === 'state' || i.authority_type === 'utility')
-      ) {
+      if ('authority' in i && i.authority && i.authority_type !== 'federal') {
```

Comment:
Thoughts?

Reply:
(Oh ugh apparently I didn't actually post this before)

Unfortunately `authority` isn't necessarily defined because `i` is a `CalculatedIncentive`, which is a supertype of `IRAIncentive`, which doesn't have the field. And if you don't have the `in` check, TS complains that `authority` doesn't exist on `CalculatedIncentive`.

I guess it would be cleaner if all the IRA incentives had `authority: null`, then this could just be `if (i.authority)`, skipping the "is federal" check. But I'd rather do that separately.

Reply:
Ah gotcha, I'd forgotten about IRAIncentives. I don't feel strongly about the cleanup – this just seemed like low-effort refactoring but ack that that's not the case.